frontend/src/tree/TreeItemContextMenu.tsx

```diff
@@ -33,9 +55,16 @@ export const TreeItemContextMenu = ({
   selection,
   setSelection,
   enterEditingMode,
-  deleteItem,
   onClose,
 }: TreeItemContextMenuProps) => {
+  const [state, setState] = useState<TreeItemContextMenuState>({ toast: 'hidden', message: null });
```

Comment:
No need to keep the state of the toast, if the message is not null than display the toast.